civiclens/regulations/models.py

```diff
@@ -73,7 +73,4 @@ class Comment(models.Model):
     restrict_reason_type = models.CharField(max_length=100, blank=True, null=True)
     submitter_rep = models.CharField(max_length=100, blank=True, null=True)
     submitter_rep_address = models.CharField(max_length=255, blank=True, null=True)
-    submitter_rep_city_state = models.CharField(max_length=100, blank=True, null=True)
-
-class Search(models.Model):
-    searchterm = models.CharField(max_length=100)
+    submitter_rep_city_state = models.CharField(max_length=100, blank=True, null=True)
```

Comment:
It looks like your end of line formatter isn't working for one reason or another.

Reply:
@abejburton fyi